accepted/uyuni-microos-salt-ssh.md

```diff
@@ -0,0 +1,85 @@
+- Feature Name: Salt SSH for transactional OSes
+- Start Date: TODO
+
+# Summary
+[summary]: #summary
+
+Uyuni users may choose to manage minions by using Salt SSH, for example due to network constraints. However, currently, systems that are based on transactional update, i.e. openSUSE MicroOS or SUSE Linux Micro (TU minions), do not support management via Salt SSH.
+
+This RFC describes improvements for Salt SSH such that Uyuni can fully control TU minions by using Salt SSH.
+
+# Motivation
+[motivation]: #motivation
+
+Uyuni users want feature parity of TU minions to non-TU minions. Currently, Uyuni users can use the Salt, Salt SSH, or Salt SSH Proxy methods to control their deployed non-TU minions. However, for TU minions, SSH management does not work.
```

Comment:
I think naming here is not the same as we are using at least on the web UI. I suppose you mean the contact methods: `Salt` (minion) / `Push via SSH` (salt-ssh) / `Push via SSH tonnel` (salt-ssh with tonnel to https of the server).